src/libraries/System.Text.Json/tests/JsonNode/JsonNodeTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace System.Text.Json.Node.Tests
+{
+    public static partial class JsonNodeTests
+    {
+        [Fact]
+        public static void JsonTypes_Deserialize()
+        {
+            var options = new JsonSerializerOptions();
```

Comment:
What's the significance of passing the `options` in these tests?

Reply:
Not needed; thanks. Remnant.